build.gradle

```diff
@@ -41,7 +41,7 @@ subprojects {
 
 allprojects { Project p ->
     group = 'eu.xenit.alfresco'
-    version = '0.3.0'
+    version = '0.2.1'
```

Comment:
Did you mean 0.3.1?
Or are we downgrading?

Reply:
Last released version is 0.2.0. These changes are bug fixes and can be done in patch versions.